docs/docsite/rst/community/development_process.rst

```diff
@@ -1,177 +1,114 @@
--  If the maintainer says “needs\_info”, the PR is labeled “needs\_info”
-   and the submitter is asked for more info.
--  If the maintainer says “needs\_revision”, the PR is labeled
-   “needs\_revision” and the submitter is asked to fix some things.
--  If the PR is in “needs\_revision/needs\_info” and has received
-   comments from the submitter:
--  If the submitter says “ready\_for\_review”, the PR is put back into
-   community\_review/core\_review and the maintainer is notified that
-   the PR is ready to be reviewed again.
--  If the PR is in “needs\_revision/needs\_info” and the submitter has
-   not responded lately:
--  The submitter is first politely pinged after two weeks, pinged again
-   after two more weeks and labeled “pending action”, and then may be
-   closed two weeks after that.
--  If the submitter responds at all, the clock is reset.
--  If the PR is in “community\_review” and the reviewer has not
-   responded lately:
--  The reviewer is first politely pinged after two weeks, pinged again
-   after two more weeks and labeled “pending\_action”, and then may be
-   reassigned to “ansible” / core\_review, or often the submitter of the
-   PR is asked to step up as a maintainer.
--  If Travis fails, or if the code is not mergable, the PR is
-   automatically put into “needs\_revision” along with a message to the
-   submitter explaining why.
-
-
-There are corner cases and frequent refinements, but this is the workflow in general. 
+Ansibullbot runs continuously. You can generally expect to see changes to your issue or pull request within thirty minutes. Ansibullbot examines every open pull request in the repositories, and enforces state roughly according to the following workflow:
+
+-  If a pull request has no workflow labels, it’s considered **new**. Files in the pull request are identified, and the maintainers of those files are pinged by the bot, along with instructions on how to review the pull request. (Note: sometimes we strip labels from a pull request to “reboot” this process.)
+-  If the module maintainer is not ``$team_ansible``, the pull request then goes into the **community_review** state.
+-  If the module maintainer is ``$team_ansible``, the pull request then goes into the **core_review** state (and probably sits for a while).
+-  If the pull request is in **community_review** and has received comments from the maintainer:
+
+   -  If the maintainer says ``shipit``, the pull request is labeled **shipit**, whereupon the Core team assesses it for final merge.
+   -  If the maintainer says ``needs_info``, the pull request is labeled **needs_info** and the submitter is asked for more info.
+   -  If the maintainer says **needs_revision**, the pull request is labeled **needs_revision** and the submitter is asked to fix some things.
+   -  If the pull request is labeled **needs_revision** or **needs_info** and has received comments from the submitter...TODO: what happens here?
```

Comment:
This sentence was left unfinsished. What happens in this case?

Reply:
This is out of date anyway and will need to be thoroughly revised when we have time. For now, let's just remove that bullet point.